model/vertex_test.go

```diff
@@ -143,3 +143,29 @@ func TestAddEdge(t *testing.T) {
 		})
 	})
 }
+
+func TestAddEdges(t *testing.T) {
+	Convey("Given a variable that represents the Vertex struct", t, func() {
+
+		v := Vertex{Type: "tesType"}
+		Convey("When 'AddEdges' is called with two edges", func() {
+			var client queryClient
+			var tstoutVID int64
+			tstoutVID = 12345
+			edge1 := EdgeWithPropsAndLabel{
+				Label:      "testLbl",
+				Id:         tstoutVID,
+				Properties: []interface{}{"tstIntrf1", 7777, "tstIntrf2", 9876},
```

Comment:
We could probably include a set property with a value of type e.g `[]int` or `[]string` and `[]interface{}` to make sure they are handled correctly.

Reply:
Properties are a list of interface and we cannot send int as interface, can we?